src/content/en/announcements/2025-04-09-release8.md

```diff
@@ -0,0 +1,42 @@
+
+### Latest software
+
+The reference implementation SCS R8 contains the latest software from the main projects. Operators have a choice though for some of the components. Most notably, they can choose to stick to old Ubuntu 22.04 LTS (over the default 24.04 LTS), OpenStack 2024.1 (over the default 2024.2) and Ceph Quincy (over Reef). SCS here follows the upstream OpenStack support for so-called "skip-level upgrade release process", which supports direct upgrades for the spring 20xx.1 releases, skipping over one 20xx.2  release. (Traditionally, skipping over releases during an upgrade was not supported.)
+
+On the container side, R8 contains the latest stable Kubernetes Cluster-API software. The included latest Cluster-API implementation for OpenStack (CAPO) now uses the OpenStack Resource Controller (ORC) which replaces the Cluster Stack Provider for OpenStack (CSPO) that was used in the SCS Cluster Stacks before. The Multi-Stage-Addons support is now used in the SCS Cluster Stacks, ensuring smooth upgrades between Kubernetes versions by sequencing upgrades of things like the CCM (Cloud Controller Manager), CNI (Cloud Networking Integration) and CSI (Cloud Storage Integration). Users have a choice between Kubernetes versions, have the option to use the latest stable versions or can upgrade existing clusters to them.
+
+For the operational tooling, SCS has also upgraded the registry (which uses harbor) and the container monitoring solution (from dNation) to the latest versions. They have been in production use by the community already for a number of weeks.
+
+[Chose one quote:]
+
+"Within the funded project period, we have built an engine for doing well-tested, predictable releases that our operator partners can use to stay up-to-date with current technology and security improvements. I'm happy to see this engine continues to be maintained and to work well, and to see the community and the technology companies step up with software engineering, resulting in solid release once more." says Kurt Garloff, S7n GmbH, former CTO of the funded SCS project.
+
+"It's good to have reference implementations of the SCS-Standards evolve to latest technologies and standards. This creates real-world usage of the standards and validates and strengthens them." says Felix Kronlage-Dammers who is heading  the Forum SCS-Standards inside the Open Source Business Alliance (OSBA).
```

Comment:
If we chose this quote, @fkr, please ensure this is using your own words.

Reply:
@garloff imho a quote from someone / an org who makes use of the ref. implementation would make more sense.

Reply:
Agreed. It puts that someone in a advantageous position though ...
Someone new we have not mentioned yet, maybe? scaleup would come to mind then.

Reply:
@garloff Excellent idea. Yes, Christoph should be able to give a suiteable quote.

Reply:
Quote from Freerk is in now.